src/ol_infrastructure/applications/micromasters/Pulumi.applications.micromasters.CI.yaml

```diff
@@ -0,0 +1,13 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-ci
+encryptedkey: AQICAHi+npazf3LfzV9oCtcYyCMYLOzaQhbo9xt6lJVVpz9tkQHmbQbdOIGG4Jt34XVtsKrHAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMzjNghqk4vTeob3lJAgEQgDt7H0yPqnjaQpouv/pngrLocorB4cYIcu+1zjTT95OxLKYWG6n4zAOokfTG64Ut0fyLkxA2EvI7vytTgg==
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-micromasters-ci.odl.mit.edu
+  consul:scheme: https
+  micromasters:db_password:
+    secure: v1:DTEttuHYUMFQ5AJM:FSsYgItu3JT8hNcO/kz/JJn3t/dSHEWl5RSNUzlErvov6GDajBte9cvhMjrWi1itHopHJCkiseHdBxjz8Iulaodo9eeHQwSMrAq3+HuWgXE=
+  micromasters:domain: ci.micromasters.mit.edu
+  micromasters:proctortrack_url: https://preproduction.verificient.com
```

Comment:
I don't think we're using that service any more but would be worth confirming.

Reply:
Could you please add that to the issue you already have open for micromasters so the devs can research this?

Reply:
We do use this service, but not on MicroMasters. It's only used for MITx Online currently.

Reply:
Peter says this is unnecessary as it's only actually used in mitxonline. Fixed.